usaspending_api/etl/subaward_etl.py

```diff
@@ -88,14 +88,13 @@ def load_subawards(submission_attributes, db_cursor):
         award_ids_to_update.add(award.id)
 
         # Find the recipient by looking up by duns
-        recipient = LegalEntity.objects.filter(recipient_unique_id=row['duns']).first()
+        recipient, created = LegalEntity.get_or_create_by_duns(duns=row['duns'])
 
-        if not recipient and row['duns']:
-            recipient, created = LegalEntity.objects.get_or_create(recipient_unique_id=row['duns'],
-                                                                   parent_recipient_unique_id=row['parent_duns'],
-                                                                   recipient_name=row["company_name"],
-                                                                   location=get_or_create_location(row, location_d1_recipient_mapper)
-                                                                   )
+        if created:
```

Comment:
We have less information coming in for the subawards, hence why it is so sparse. We can try and pull more information by joining other tables in the future if we need to.